qa/tasks/cephfs/test_volumes.py

```diff
@@ -3867,6 +3867,77 @@ def test_clone_subvolume_info(self):
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def test_subvolume_clone_info_and_no_pending_clones(self):
```

Comment:
s/clone_info/snapshot_info ?

Reply:
Suggest adding doc strings to test methods briefly describing the test